### ISSUE TITLE
How to fix `ERROR: for db Container command '/docker-entrypoint1.sh' could not be invoked` on Windows

### ISSUE THREAD

**Issue by pdaoust** [mattermost/mattermost-docker#20]
I thought I'd try out this Docker setup, but after all the images are built, I get this message:

```
ERROR: for db Container command '/docker-entrypoint1.sh' could not be invoked.
```

I don't get any debugging messages, but a bit of sleuthing discovered that this is caused by a couple Docker-and-Git-on-Windows issues:

* Windows doesn't support the executable bit, and
* Windows Git clients are usually set up with `autocrlf`, which adds a `^M` to the end of the `#!/bin/bash` line in `db/docker-entrypoint1.sh`.

So, for Windows users I'd recommend the following:

* In the repo's directory, run `git config core.autocrlf false` and checkout the code again (using `git reset --hard HEAD` or whatever you prefer).
* In `db/docker-entrypoint1.sh`, add the line `RUN chmod +x /docker-entrypoint1.sh` after the line `COPY docker-entrypoint1.sh /`.

These two things allowed me to run Mattermost without troubles on Windows!

**Comment by npcode** [comment]
Thanks for the reporting! 

> In the repo's directory, run git config core.autocrlf false and checkout the code again (using git reset --hard HEAD or whatever you prefer).

I'm considering to push images to dockerhub to avoid this problem.

> In db/docker-entrypoint1.sh, add the line RUN chmod +x /docker-entrypoint1.sh after the line COPY docker-entrypoint1.sh /.

I fixed it as you said at 808f5af6698fbbb39a73229a4445a19da3507527 and will merge it into master soon.

**Comment by andymac4182** [comment]
Also need something similar to https://gist.github.com/andymac4182/0b82297a16f5c7ad1eaa72f7b623b5a4 which is what we use for our docker repos with windows.
